frontend/src/pages/AboutUs.jsx

```diff
@@ -101,16 +99,16 @@ function AboutUs() {
             isClicked.Theo ? "displayOn" : "displayOff"
           }`}
         >
-          Théo, chaman des quatre terres, druide de l’ancien monde, grand PO
-          suprême, concquérant des cinq royaumes, élu parmis les élus, artisan
-          du design web Il accorde une attention méticuleuse aux détails et
-          s'efforce de créer des interfaces utilisateur élégantes et intuitives.
-          Avec un sens aigu de l'esthétique, il jongle avec les palettes de
-          couleurs, les typographies et les éléments graphiques pour donner vie
-          à des sites web qui captivent l'œil. Chaque pixel est soigneusement
-          placé, et chaque transition est pensée avec minutie. Théo sait que la
-          beauté et la convivialité vont de pair, et il s'efforce de créer des
-          expériences web qui laissent une impression durable.
+          Théo, shaman of the four lands, druid of the old world, great PO
+          supreme, conqueror of the five kingdoms, chosen among the chosen,
+          artisan of web design He pays meticulous attention to details and
+          strives to create elegant and intuitive user interfaces. With a keen
+          sense of aesthetics, he juggles with palettes of colors, typographies
+          and graphic elements to bring life to websites that captivate the eye.
+          Every pixel is carefully placed, and each transition is carefully
+          thought out. Theo knows that beauty and friendliness go hand in hand,
+          and he strives to create web experiences that leave a lasting
+          impression.
```

Comment:
Cadeau => Théo, -shaman of the four lands, druid of the ancient world, grand supreme UX designer, conqueror of the five kingdoms, chosen among the chosen, artisan of web design-, pays meticulous attention to details and strives to create elegant and intuitive user interfaces. With a keen sense of aesthetics, he juggles color palettes, typography, and graphic elements to bring life to websites that captivate the eye. Every pixel is carefully placed, and every transition is thought through meticulously. Théo understands that beauty and user-friendliness go hand in hand, and he works to create web experiences that leave a lasting impression.